### PR TITLE
Add a badge for Go Report Card

### PR DESCRIPTION
**Pull request by mingrammer**
I've added a badge for [Go Report Card](https://goreportcard.com/) which reports the quality of an open source go project using `gofmt`, `go vet` and so on. 

This allows users to see at a glance how well the source code is formatted.

### CODE REVIEW

**Comment by kortschak** [comment]
Note that the incorrect vet report is a [known issue](https://github.com/gojp/goreportcard/issues/113).

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Gonum [![Build Status](https://travis-ci.org/gonum/gonum.svg?branch=master)](https://travis-ci.org/gonum/gonum) [![Coverage Status](https://coveralls.io/repos/gonum/gonum/badge.svg?branch=master&service=github)](https://coveralls.io/github/gonum/gonum?branch=master) [![GoDoc](https://godoc.org/gonum.org/v1/gonum?status.svg)](https://godoc.org/gonum.org/v1/gonum)
+# Gonum [![Build Status](https://travis-ci.org/gonum/gonum.svg?branch=master)](https://travis-ci.org/gonum/gonum) [![Coverage Status](https://coveralls.io/repos/gonum/gonum/badge.svg?branch=master&service=github)](https://coveralls.io/github/gonum/gonum?branch=master) [![GoDoc](https://godoc.org/gonum.org/v1/gonum?status.svg)](https://godoc.org/gonum.org/v1/gonum) [![Go Report Card](https://goreportcard.com/badge/github.com/gonum/gonum)](https://goreportcard.com/report/github.com/gonum/gonum)
```
Comment (kortschak) [inline README.md]: This should really point to the canonical import, but goreportcard does not correctly resolve links to the backing repository when this is done https://github.com/gojp/goreportcard/issues/192

Comment (btracey) [inline README.md]: Ah, that explains why when I ran it on the "go get" path (what the documentation says), I got 0% for all the checks. In other news, the number of false positives is annoying (Statuser is not a misspelling of Stature, it's actually following Go convention)

Comment (btracey) [inline README.md]: For that one at least: https://github.com/client9/misspell/issues/119

**Comment by mingrammer** [comment]
Ye, I'll add myself to A+C soon.

**Pull request merged**